polaris-react/src/components/Banner/Banner.scss

```diff
@@ -1,50 +1,9 @@
 @import '../../styles/common';
 
-@mixin banner-variants($in-page) {
```

Comment:
Not using this mixin anymore as it just made the file confusing. I put the appropriate styles in the `withinPage` and `withinContentContainer` classes since thats the only place this mixin was being called. Shared styles went to `.Banner`. `.Banner`, `.withinPage`, and `.withinContentContainer` are all only applied to the same wrapping div